packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -213,26 +213,22 @@ class LayoutBuilder extends ConstrainedLayoutBuilder<BoxConstraints> {
 class _RenderLayoutBuilder extends RenderBox with RenderObjectWithChildMixin<RenderBox>, RenderConstrainedLayoutBuilder<BoxConstraints, RenderBox> {
   @override
   double computeMinIntrinsicWidth(double height) {
-    assert(_debugThrowIfNotCheckingIntrinsics());
-    return 0.0;
+    return child != null ? child.computeMinIntrinsicWidth(height) : 0.0;
   }
 
   @override
   double computeMaxIntrinsicWidth(double height) {
-    assert(_debugThrowIfNotCheckingIntrinsics());
-    return 0.0;
+    return child != null ? child.computeMaxIntrinsicWidth(height) : 0.0;
   }
 
   @override
   double computeMinIntrinsicHeight(double width) {
-    assert(_debugThrowIfNotCheckingIntrinsics());
-    return 0.0;
+    return child != null ? child.computeMinIntrinsicHeight(width) : 0.0;
   }
 
   @override
   double computeMaxIntrinsicHeight(double width) {
-    assert(_debugThrowIfNotCheckingIntrinsics());
-    return 0.0;
+    return child != null ? child.computeMaxIntrinsicHeight(width) : 0.0;
```

Comment:
This returns data that may be outdated:

computeMaxIntrinsicHeight/getMaxIntrinsicHeight is called before the child is re-layed out. So if the max intrinsic height of the child has changed since the last layout cycle, this will return the old value, which is incorrect.

https://github.com/flutter/flutter/blob/55fd5f1561b4590c20d5f51ae1064e62080ed74c/packages/flutter/lib/src/rendering/proxy_box.dart#L697-L710